daisy_workflows/build-publish/rhui/cds.publish.json

```diff
@@ -1,26 +1,25 @@
 {
   "Name": "cds",
   {{$work_project := printf "%q" "gce-image-builder" -}}
-  {{$endpoint := `"https://www.googleapis.com/compute/alpha/projects/"` -}}
   {{$delete_after := `"24h*30*6"` -}}
-  {{if eq .environment "prod" -}}
-  "WorkProject": {{$work_project}},
-  "PublishProject": "rhel-cloud",
-  "ComputeEndpoint": {{$endpoint}},
+
+  "ComputeEndpoint": "https://www.googleapis.com/compute/alpha/projects/",
+  {{if eq .environment "test" -}}
+  "PublishProject": "google.com:rhel-infra",
+  {{- else if eq .environment "prod" -}}
+  "PublishProject": "google.com:rhel-infra",
   {{- else -}}
```

Comment:
did you want the projects to be the same for test and prod?

Reply:
yep! will change when the project really becomes 'prod' (at which time we can see what it means to publish 'test')